docs/general/dql/selects.rst

```diff
@@ -622,9 +623,9 @@ This behaviour applies to:
     <indices.query.bool.max_clause_count>` setting as appropriate on each node.
 
 
-.. _sql_dql_nonscalar:
+.. _sql_dql_compound:
 
-Nonscalar data types
+Compound data types
```

Comment:
Maybe one of the last spots where there is still "compound" mentioned?

Reply:
good catch! thx